setup/pi/configure-web.sh

```diff
@@ -28,6 +28,15 @@ mkdir /var/www/html/TeslaCam
 cp -rf "$SOURCE_DIR/teslausb-www/teslausb.nginx" /etc/nginx/sites-available
 ln -sf /etc/nginx/sites-available/teslausb.nginx /etc/nginx/sites-enabled/default
 
+# Setup /etc/nginx/.htpasswd if user requested web auth, otherwise disable auth_basic
+if [ "${WEB_AUTH_ENABLED:-false}" = "true" ] && [ -n "${WEB_USERNAME}" ] && [ -n "${WEB_PASSWORD}" ]
```

Comment:
instead of adding the WEB_AUTH_ENABLED variable, why not just check like this: `[ -n "${WEB_USERNAME:-}" ]` ?

Reply:
I agree, I think that makes more sense, thank you.